instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/cache/WeakLockFreeCache.java

```diff
@@ -5,7 +5,7 @@
 
 package io.opentelemetry.instrumentation.api.cache;
 
-import io.opentelemetry.instrumentation.api.cache.weaklockfree.WeakConcurrentMap;
+import io.opentelemetry.context.internal.shaded.WeakConcurrentMap;
```

Comment:
This import looks wrong. I guess sdk also has a copy of weaklockfree

Reply:
It is wrong. Thanks for noticing